.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug Report
+about: Create a report to help us fix bugs in Betaflight Blackbox Explorer
+labels: "Template: Bug"
+---
+<!-- This is a template that you must fill. If not, the message will be closed. So don't erase any subtitle in this template (they start with ###)
+and complete all of them -->
+
+### Describe the bug
+<!-- A clear and concise description of what the bug is. -->
+
+### To Reproduce
+<!-- Steps to reproduce the behavior -->
+
+### Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+### Setup
+<!-- Specify the version of Betaflight Blackbox Explorer (as displayed in the app) -->
+- Blackbox Explorer version:
+
+<!-- If this bug is related to a specific flight controller and/or configuration please specify it here. Create a diff and post it here in a code block. Put (three backticks) at the start and end of the diff block (instructions  on how to do a diff: https://oscarliang.com/use-diff-not-dump-betaflight/) -->
```

Comment:
This line seems relevant to a flight controller or firmware issue.  I wonder if here, for logs it might be more useful to ask the user to post the log header, or a link to the log  file itself with which they are having a problem?

Reply:
Yes that makes sense, I updated the PR to specify a log instead.